src/new-components/Faq.vue

```diff
@@ -0,0 +1,302 @@
+  min-width: 220px;
+  padding-top: 3%;
+  cursor: pointer;
+  background:rgba(0,0,0,0.2); 
+}
+
+.category-bubble {
+  text-align: left;
+  font-size: 20px;
+  padding: 8% 10%;
+  transition: 0.1s ease-in-out;
+  margin-left: 10px; 
+  margin-right: 10px; 
+}
+
+.selected {
+  font-size: 20px;
+  background:rgba(0,0,0,0.4); 
+  clip-path: polygon(0 0, 100% 0, 100% 70%, 93% 100%, 0 100%, 0% 50%); 
+}
+
+.faq-area {
+  padding-left: 4%;
+  padding-right: 8%;
+  padding-top: 2%;
+  width: 100%;
+  overflow: auto;
+  position: relative;
+  z-index: 40;
+  background: rgba(255, 255, 255, 0.15);
+  text-align: center;
+  padding-bottom: 2vh;
+}
+
+.mobile-faq-area {
+  max-height: 300px;
+  overflow: auto;
+  display: none;
+}
+
+.faq-area {
+  display: block;
+}
+
+.question-box {
+  font-size: 19px;
+  padding: 3% 0;
+  padding-bottom: 1%;
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+}
+
+.question {
+  cursor: pointer;
+  font-weight: 600;
+}
+
+.answer-box {
+  max-height: 500px;
+  overflow: hidden;
+  padding: 1% 0;
+  font-size: 16px;
+}
+
+.open-enter-active,
+.open-leave-active {
+  transition: max-height 0.2s;
+}
+
+.open-enter,
+.open-leave-to {
+  max-height: 0;
+}
+
+.arrow {
+  -moz-transition: all 0.2s linear;
+  -webkit-transition: all 0.2s linear;
+  transition: all 0.2s linear;
+  display: inline-block;
+  float: right;
+}
+
+.rotate {
+  -ms-transform: rotate(180deg);
+  -moz-transform: rotate(180deg);
+  -webkit-transform: rotate(180deg);
+  transform: rotate(180deg);
+}
+
+@media only screen and (max-width: 700px) {
+  .faq-container {
+    display: flex;
+    flex-direction: column;
+    clip-path: polygon(0 0, 100% 0, 100% 100%, 100% 100%, 0 100%, 0% 50%); 
+    height: 700px; 
+    width: 90%; 
+    background: none;
+  }
+  .faq h1 {
+    padding: 0px;
+  }
+  .categories {
+    display: flex;
+    flex-direction: column;
+    padding: 0px;
+    background:rgba(0,0,0,0.4);
+  }
+  .category-bubble {
+    text-align: center;
+    font-size: 22px;
+    padding: 2%;
+    padding-bottom: 5%;
+    padding-top: 5%;
+  }
+  .selected {
+    font-size: 30px;
+    font-weight: 800;
+    text-align: center;
+    text-decoration: none;
+    font-size: 24px; 
+    font-weight: 800;
+    opacity: 0.99;
+    padding: 10px 20px;
+    color: rgba(255, 255, 255, 1);
+    background-color: rgba(255, 255, 255, 0.2);
+    transition: 0.1s ease-in-out;
+    cursor: pointer;
+    z-index: 10000;
+    margin-top: 10px; 
+    margin-bottom: 10px; 
+    clip-path: polygon(0 0, 100% 0, 100% 85%, 90% 100%, 0 100%, 0% 50%);
+  }
+  .answer-box {
+    font-weight: 400;
+  }
+  #desktop-faq {
+    display: none;
+  }
+  #mobile-faq {
+    display: block;
+  }
+  .mobile-faq-area {
+    display: block;
+    padding: 3%;
+  }
+  .faq-area {
+    display: none;
+  }
+  .open-enter-active,
+  .open-leave-active {
+    transition: max-height 0.3s;
+  }
+}
+::-webkit-scrollbar {
+  width: 0px; /* Remove scrollbar space */
```

Comment:
Dunno if we should be making the scrollbar invisible when our content overflows. I'm playing around with the component on mobile, I have a bunch of questions for a category open, and it's kind of odd to orient myself on how far I am from the top/bottom of the category.

If we received a spec to hide the scroll bar, then you can ignore this (or if @enricoflorentino doesn't have an issue with the lack of a scroll bar). O/w, perhaps we can style the scroll bar to blend in with the component

Reply:
Yeah no worries lol. I think I'll leave it as is for now, and see if we're tryna add a scrollbar or not once we get the mobile specs.